packages/services/src/validateContext.ts

```diff
@@ -233,3 +234,31 @@ export const validateDigitalFacilityRecordContext = (
 
   return { ok: true, value: context };
 };
+
+export const validateDigitalConformityCredentialContext = (
+  context: IDigitalConformityCredentialContext,
+): Result<IDigitalConformityCredentialContext> => {
+  const validationResult = checkContextProperties(context);
+  if (!validationResult.ok) return error(validationResult.value);
+
+  if (_.isEmpty(context.vckit.vckitAPIUrl)) return error('Invalid vckitAPIUrl');
+  if (_.isEmpty(context.vckit.issuer)) return error('Invalid issuer');
+
+  if (_.isEmpty(context.digitalConformityCredential)) return error('Invalid digitalConformityCredential context');
+  if (_.isEmpty(context.digitalConformityCredential.context))
+    return error('Invalid digitalConformityCredential context');
+  if (_.isEmpty(context.digitalConformityCredential.type)) return error('Invalid type');
+  if (_.isEmpty(context.digitalConformityCredential.dlrLinkTitle)) return error('Invalid dlrLinkTitle');
+  if (_.isEmpty(context.digitalConformityCredential.dlrVerificationPage)) return error('Invalid dlrVerificationPage');
+  if (_.isEmpty(context.digitalConformityCredential.dlrIdentificationKeyType))
+    return error('Invalid dlrIdentificationKeyType');
+
```

Comment:
Should we include a check for the `renderTemplate`? If that's a optional field, please skip this review.

Reply:
Optional